pkg/test/xds/secrets.go

```diff
@@ -54,34 +54,34 @@ func get(meshes []*core_mesh.MeshResource) (*core_xds.IdentitySecret, map[string
 		},
 	}
 
-	return identitySecret, cas, allInOne, nil
+	return identitySecret, cas, allInOne
 }
 
 func (*TestSecrets) GetForZoneEgress(
 	_ *core_mesh.ZoneEgressResource,
 	mesh *core_mesh.MeshResource,
 ) (*core_xds.IdentitySecret, *core_xds.CaSecret, error) {
-	identity, cas, _, err := get([]*core_mesh.MeshResource{mesh})
+	identity, cas, _ := get([]*core_mesh.MeshResource{mesh})
 
-	return identity, cas[mesh.GetMeta().GetName()], err
+	return identity, cas[mesh.GetMeta().GetName()], nil
 }
 
 func (*TestSecrets) GetForDataPlane(
 	_ *core_mesh.DataplaneResource,
 	mesh *core_mesh.MeshResource,
 	meshes []*core_mesh.MeshResource,
 ) (*core_xds.IdentitySecret, map[string]*core_xds.CaSecret, error) {
-	identity, cas, _, err := get(append([]*core_mesh.MeshResource{mesh}, meshes...))
-	return identity, cas, err
+	identity, cas, _ := get(append([]*core_mesh.MeshResource{mesh}, meshes...))
+	return identity, cas, nil
```

Comment:
why do we need an err if we always return nil?

Reply:
It's an interface